tiledb/sm/query/strategy_base.cc

```diff
@@ -79,6 +79,10 @@ stats::Stats* StrategyBase::stats() const {
 void StrategyBase::get_dim_attr_stats() const {
   for (const auto& it : buffers_) {
     const auto& name = it.first;
+    if (!array_schema_.is_field(name)) {
+      continue;
+    }
```

Comment:
It just occurred to me after seeing this a few times that this is likely because the aggregate buffer names are mixed with field names. I think this is just further proof that mixing these two pools of buffers is a bad idea.

Reply:
I'll try to split them up and see what it yields.